data/txHandler.go

```diff
@@ -188,7 +189,7 @@ func (handler *TxHandler) backlogWorker() {
 func (handler *TxHandler) asyncVerifySignature(arg interface{}) interface{} {
 	tx := arg.(*txBacklogMsg)
 	for _, txn := range tx.unverifiedTxGroup {
-		tx.verificationErr = txn.Verify(tx.spec, tx.proto)
+		tx.verificationErr = verify.Txn(&txn, tx.spec, tx.proto)
```

Comment:
does the variable content might get modified by the next for-loop iteration before the work on the verification is complete ? If so, it might be a data-race use case.